http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1133,18 +1149,32 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         )
       )(app => app.namespaceName.asRight[Throwable])
 
+      potentialNewChart =
```

Comment:
I see, we're building the actual chart+version here and persisting to the database. So the APP table has
* appType: ALLOWED
* chart: the-actual-chart:version

Reply:
yea exactly...what's persisted to DB will be the truth...what's in config is not entirely the final value